processing_handler.go

```diff
@@ -91,6 +91,14 @@ func setCacheControl(rw http.ResponseWriter, force *time.Time, originHeaders map
 	}
 }
 
+func setLastModified(rw http.ResponseWriter, originHeaders map[string]string) {
+	if originHeaders != nil {
```

Comment:
We should probably add a `config.LastModifiedEnabled` check here. If we set the `Last-Modified` header, a client may use it for its cache revalidation. And that may lead to unpredictable behavior. Better safe than sorry.

BTW, it's safe to get access to nil map's keys, so we don't need the `originHeaders != nil` check: https://goplay.tools/snippet/3OsmlN7HqxD

Reply:
Fixed!